go.mod

```diff
@@ -115,6 +115,7 @@ require (
 	golang.org/x/sync v0.0.0-20190423024810-112230192c58
 	golang.org/x/sys v0.0.0-20190730183949-1393eb018365
 	golang.org/x/text v0.3.2 // indirect
+	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
```

Comment:
why is it changed?

Reply:
because i'm using rate.Limiter from "golang.org/x/time/rate" package